src/components/LibraryItem.tsx

```diff
@@ -258,6 +268,14 @@ export class LibraryItem extends React.Component<LibraryItemProps, LibraryItemSt
         }
     }
 
+    // Collapse all child items when one of the child items is expanded
+    onItemWillExpand() {
```

Comment:
Can we rename as I originally recommended? This is at the parent level and we need to make that clear.